src/scripts/site.js

```diff
@@ -647,12 +648,12 @@ var AppInfo = {};
             inputManager: "scripts/inputManager",
             datetime: "scripts/datetime",
             globalize: "scripts/globalize",
+            dfnshelper: "scripts/dfnshelper",
```

Comment:
It'd rename the module to something more explicit and generic, like `DateLocaleHelper`.

Reply:
@MrTimscampi I agree that dfnshelper it's not the best name, but it's short and you can correlate it to date-fns. ``DateLocaleHelper`` doesn't mention at all date-fns, so it might be confusing in the long-term. It  only handles the locale objects of date-fns, nothing to do with globalize or anything else related to localization, so I think it's a good idea to leave some kind of mention to date-fns